docs/scoring.md

```diff
@@ -0,0 +1,39 @@
+# Goal
+
+The goal of this document is to explain how scoring on Lighthouse works, and how to understand what to do to improve your
+Lighthouse score. 
+
+# What metrics does Lighthouse measure?
+
+Lighthouse primarily measures the following metrics: 
+
+- First meaningful paint: first meaningful paint is defined as when the browser first puts any “meaningful” element/set of “meaningful” elements  on the screen. What is meaningful is determined from a series of heuristics. 
```

Comment:
do we want to link out to each individual audit documentation that explains this with links to all the relevant explainers and such?

Reply:
done!